### PR TITLE
Move sig-cli gce etcd3 tests into their own job

### PR DESCRIPTION
**Pull request by jennybuckley**
also send alert emails to the mailing list 'kubernetes-sig-cli-oncall+alerts@gmail.com'

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -3689,6 +3689,23 @@
       "UNKNOWN"
     ]
   },
+  "ci-kubernetes-e2e-gci-gce-etcd3-sig-cli": {
```
Comment (BenTheElder) [inline jobs/config.json]: @krzyzacy perhaps we should drop these `etcd3`s since that's de-facto now?

Comment (krzyzacy) [reply jobs/config.json]: hummmm, I think the default gci-gce job is still testing against etcd2 :joy:

Comment (BenTheElder) [reply jobs/config.json]: maybe we should fix that too then?

Comment (krzyzacy) [reply jobs/config.json]: cc @msau42 @saad-ali - do we still care about etcd2 on master? :see_no_evil:

**Comment by krzyzacy** [comment]
`also send alert emails to the mailing list 'kubernetes-sig-cli-oncall+alerts@gmail.com'` huh? you want to add to your testgrid config?

**Comment by jennybuckley** [comment]
@krzyzacy Oops, looks like that part didn't get added to the commit for some reason. I've updated the pr

File: jobs/config.json (hunk 1 above)
Comment (msau42) [reply jobs/config.json]: cc @kubernetes/sig-api-machinery-pr-reviews for etcd2 questions

**Comment by krzyzacy** [comment]
/lgtm

where's the `sig/api-machinery` label come from?

**Comment by jennybuckley** [comment]
@krzyzacy i think from the comment @msau42 made in the discussion about etcd2

**Comment by krzyzacy** [comment]
ah missed that, thanks!

**Pull request merged**